group_vars/all/vars.yml

```diff
@@ -125,3 +125,5 @@ db_backup_filename: "{{ timestamp }}_{{ application_db_name }}.sql.bz2"
 db_backup_path: "{{ backup_dir }}{{ db_backup_filename }}"
 media_backup_filename: "{{ timestamp }}_{{ app_name }}_media.tar.gz"
 media_backup_path: "{{ backup_dir }}{{ media_backup_filename}}"
+## nfs
+nfs_server_staging: "128.112.203.82"
```

Comment:
@kayiwa do you want to define this in `hosts` and pull it in from there? or does it need to be referenced by IP?

Reply:
another thought: we could define this in `group_vars/qa/vars.yml` as just `nfs_server` and use the same variable with different values for staging/prod hosts